autoscale/src/main/java/com/sequenceiq/periscope/config/DatabaseConfig.java

```diff
@@ -92,7 +102,33 @@ private Properties jpaProperties() {
         properties.setProperty("hibernate.hbm2ddl.auto", hbm2ddlStrategy);
         properties.setProperty("hibernate.show_sql", "false");
         properties.setProperty("hibernate.dialect", "org.hibernate.dialect.PostgreSQLDialect");
+        properties.setProperty("hibernate.default_schema", StringUtils.isEmpty(dbSchemaName) ? "public" : dbSchemaName);
         return properties;
     }
 
+    private String createDbUrl() {
+        String url;
+        if (!StringUtils.isEmpty(dbSchemaName)) {
```

Comment:
Because public is the default schema if we set it as default that part is not necessary any more.